JsonFx/JsonFx.Json.UnitTests/JsonFx.Json.UnitTests.csproj

```diff
@@ -18,7 +18,7 @@
     </SccAuxPath>
     <SccProvider>
     </SccProvider>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>

```

Comment:
Does this work? When I was building the v4 csproj files in the past the structure would look very different - not having to list every file for example...

Reply:
At first sight it looks like it, the tests run and pass. This was to make the unit tests depend on the netstandard2.0 version, as that seems to be the way forward. There may be other ways to do it that I don't know about though.

Reply:
Does a .netstandard2.0 DLL build okay from this?

Reply:
Yeah, looks like it. The tests project is now using the netstandard version as a dependency. Also successfully used this dll in netstandard and netcore test projects.